lib/livebook/zta/teleport.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Livebook.ZTA.Teleport do
+  use GenServer
+  require Logger
+
+  defstruct [:name, :req_options, :jwks]
+
+  @renew_afer 24 * 60 * 60 * 1000
+  @fields %{"sub" => :id, "username" => :username}
+  @assertion "teleport-jwt-assertion"
+  @well_known_jwks_path "/.well-known/jwks.json"
+
+  def start_link(opts) do
+    url = String.trim(opts[:identity][:key], "/")
```

Comment:
the teleport web server doesn't like when there are double `/` in the url and give 404!

Reply:
@slashmili use `URI.append_path` and that should handle duplicates.